CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # AppMap Plugin Changelog
 
+## [Unreleased]
```

Comment:
Instead of marking this "Unreleased", can you bump the version to 0.7.6, please?

Reply:
I've pushed the change.

Reply:
Thanks!